### PR TITLE
add route rule and destination policy to httpbin sample to be used wi…

### PR DESCRIPTION
**Pull request by christian-posta**
…th circuit breaking task from the docs https://github.com/istio/istio.github.io/issues/872

### CODE REVIEW

File: samples/httpbin/destinationpolicies/httpbin-circuit-breaker.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: config.istio.io/v1beta1
```
Comment (rshriram) [inline samples/httpbin/destinationpolicies/httpbin-circuit-breaker.yaml]: v1beta1 ? isn't it v1alpha1 ?

File: samples/httpbin/routerules/httpbin-v1.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: config.istio.io/v1alpha2
```
Comment (rshriram) [inline samples/httpbin/routerules/httpbin-v1.yaml]: this is v1alpha1

**Comment by christian-posta** [comment]
doh! thanks @rshriram i've updated

File: samples/httpbin/sample-client/gobench-deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: gobench
+        version: v1
+    spec:
+      containers:
+      - image: piotrminkina/gobench:latest
```
Comment (ldemailly) [inline samples/httpbin/sample-client/gobench-deploy.yaml]: we chatted offline and agreed to replace this unknown image by fortio

**Pull request merged**